erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md

```diff
@@ -0,0 +1,54 @@
+<!-- add-breadcrumbs -->
+
+# Exotel Integration
+
+This integration enables you to integrate Exotel with your ERPNext system, if you are using Exotel for business to capture the leads with their phone number directly in your ERPNext account.
+
+## Features
+
+1. Track incoming calls in your ERPNext account.
+2. Shows lead/customer information pop-up to employees when an incoming call is received.
+
+## How to setup
+
+### 1. Setup in your Exotel account for call popup
+
+- Login to your Exotel account and go to App Bazar
+- Create a new App for a new flow.
+- Setup the flow as you wish it to be.
+- In your connect API under "Create popup..." and paste following URL:
+`https://<your-site>/api/method/erpnext.erpnext_integrations.exotel_integration.handle_incoming_call`
+
+<img class="screenshot" alt="Connect Applet" src="{{docs_base_url}}/assets/img/erpnext_integrations/exotel_integration/connect_applet.png">
```

Comment:
```suggestion
![Connect Applet](/docs/assets/img/erpnext_integrations/exotel_integration/connect_applet.png)
```

Reply:
he's just showing off that he's a web dev. "HTML" /jk
but seriously md image tags are simpler